samples/net/ptp/slave/prj.conf

```diff
@@ -0,0 +1,22 @@
+# Enable networking
+CONFIG_NETWORKING=y
+CONFIG_NET_LOG=y
+
+# Network settings
+CONFIG_NET_CONFIG_SETTINGS=y
+CONFIG_NET_CONFIG_NEED_IPV4=y
+CONFIG_NET_CONFIG_MY_IPV4_ADDR="192.0.2.1"
+CONFIG_NET_CONFIG_PEER_IPV4_ADDR="192.0.2.2"
+
+# Simple log
+CONFIG_LOG=y
+
+# PTP
+CONFIG_NET_PTP=y
+CONFIG_NET_PTP_SLAVE=y
+CONFIG_NET_PTP_LOG_LEVEL_DBG=y
+
+# Socket APIs
+CONFIG_NET_SOCKETS=y
+CONFIG_NET_SOCKETS_PACKET=y
+CONFIG_NET_SOCKETS_POSIX_NAMES=y
```

Comment:
Hmm, is socket API used anywhere here, I didn't see any use in this PR? If not, please remove those configs.